src/Datadog.Trace/Conventions/DatadogOutboundHttpConvention.cs

```diff
@@ -33,5 +33,10 @@ public Scope CreateScope(OutboundHttpArgs args, out HttpTags tags)
             tags.SetAnalyticsSampleRate(integrationId, _tracer.Settings, enabledWithGlobalSetting: false);
             return scope;
         }
+
+        public TraceId GenerateNewTraceId()
+        {
+            return TraceId.CreateRandom64Bit();
+        }
```

Comment:
This should NOT be in `DatadogOutboundHttpConvention` class.
I was considering to make a `Convention` facade class and it can be introduced in this PR.
More: 
- https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/63#discussion_r585289877
- https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/63#discussion_r587331037

Reply:
Yes, that's what I thought. I will create a separate interface for it for now, and add a field to Tracer class.

Reply:
Done.

Reply:
Resolve